edgekv/lib/README.md

```diff
@@ -22,6 +22,8 @@ The edgekv.js file must be included alongside `edgekv_tokens.js` which includes
 	 * @param {string} [$0.group] the default group to use for all GET, PUT, and DELETE operations
 	 * 		Group must be 128 characters or less, consisting of A-Z a-z 0-9 _ or -
 	 * @param {number} [$0.num_retries_on_timeout=0] the number of times to retry a GET requests when the sub request times out
+	 * @param {object} [$0.ew_request] the EdgeWorker request object passed into the handler
+	 * @param {boolean} [$0.sandbox_fallback=false] whether to fallback on staging data if nothing is found in the sandbox during read operations
```

Comment:
Suggested edit:
"Optional parameter that can be set to true to to fallback on staging data if nothing is found in the sandbox during read operations, otherwise the read will return null or the specified default value"